google-beta/resource_compute_instance_group_manager.go

```diff
@@ -345,38 +351,45 @@ func flattenFixedOrPercent(fixedOrPercent *computeBeta.FixedOrPercent) []map[str
 func getManager(d *schema.ResourceData, meta interface{}) (*computeBeta.InstanceGroupManager, error) {
 	config := meta.(*Config)
 
-	project, err := getProject(d, config)
+	id, err := parseInstanceGroupManagerId(d.Id())
```

Comment:
nit: `id` is almost always a string elsewhere in the code base. Since this is a struct, maybe rename to something more unique?